admins/pageflow/entry.rb

```diff
@@ -9,24 +9,33 @@ module Pageflow
         entry_publication_state_indicator(entry)
       end
       column :title, sortable: 'title' do |entry|
-        link_to(entry.title, admin_entry_path(entry))
+        if entry.title.blank?
+          link_to(I18n.t('pageflow.admin.entries.entry_number') + ' ' + entry.id.to_s,
+                  admin_entry_path(entry))
+        else
+          link_to(entry.title, admin_entry_path(entry))
+        end
```

Comment:
Good idea. Apart from the title, the `link_to` call is the same though in both cases. Let's extract an `admin_entry_title` helper method to `Pageflow::Admin::EntriesHelper`. Also let's use i18n interpolation instead of explicit concatenation: `I18n.t('pageflow.admin.entries.default_title, id: entry.id)`. The translation can then be `"Entry %id"` for example.